lib/src/Pos/Web/Server.hs

```diff
@@ -84,24 +91,87 @@ application = do
     server <- servantServer
     return $ serve nodeApi server
 
-serveImpl
-    :: (HasConfiguration, MonadIO m)
-    => m Application
+type TLSServer m =
+    m Application
```

Comment:
I usually find these kinds of type synonyms to provide more indirection than readability -- I almost always need to jump-to-definition to find out *what* a `TLSServer` is. Could you write a comment explaining why you use this alias?

Reply:
Not sure where this comes from. Might have picked up somewhere or from before and it stayed as is. 
I do agree with you though. Will change :+1: